examples/simple/client/routes/index.ts

```diff
@@ -1,9 +1,9 @@
-import { readFileSync } from 'fs'
-import { join } from 'path'
+import { readFileSync } from "fs";
+import { join } from "path";
```

Comment:
I think your vscode prettier is not in sync with the repository or you aren't running the husky pre-hook.

Reply:
According to our [prettier config](https://github.com/Aghassi/bazel-module-federation/blob/main/prettier.config.js#L6), this is the correct quote change.

Reply:
Oh weird haha. I’m so used to whatever I wrote or what I have at work. All good then.